app/models/mixins/ownership_mixin.rb

```diff
@@ -66,8 +66,16 @@ def conditions_for_owned_or_group_owned(user_or_group = nil)
       end
     end
 
-    def user_or_group_owned(user = nil)
-      where(conditions_for_owned_or_group_owned(user)).to_a
+    def user_or_group_owned(user, miq_group)
+      if user && miq_group
+        where("evm_owner_id" => user.id).or(where("miq_group_id" => miq_group.id))
```

Comment:
@chessbyte it was failing because these were coming off of a join
Will change back and see if I can get the specs to still pass